solidity/dashboard/README.md

```diff
@@ -75,6 +75,13 @@ When you don’t want to use the local version of `@keep-network/keep-core` anym
 
 A new version of staking dApp is automatically deployed to `keep-dev` internal testnet after each `master` merge. dApp can be accessed at [https://dashboard.dev.keep.network/](https://dashboard.dev.keep.network/) and requires an initial setup in MetaMask before the first use. All the setup described below has to be done only one time. 
 
+### Ropsten testnet
+
+Change:
+In `solidity/dashboard/src/connectors/utils.js`
+- getChainId() return value to 3. As indicated, private chains ia the default, so you need to change if you use a different one (Ropsten = 3).
```

Comment:
`getChainId` is deriving from the artifact dependencies, so it should not be hardcoded. It will be correct as long as the `package.json` points to the correct artifacts.

Reply:
Here's a snippet of `getChainId()` [src](https://github.com/keep-network/keep-core/blob/master/solidity/dashboard/src/connectors/utils.js#L25-L32)
```javascript
export const getChainId = () => {
  if (process.env.NODE_ENV === "development") {
    // private chains (default), change if you use a different one
    return 1337
  }
  // For KEEP internal testnet, ropsten and mainnet `chainId == networkId`
  return Number(getFirstNetworkIdFromArtifact())
}
```
When we run with `npm start` it runs in development mode, hence NODE_ENV would cause this to return the hard-coded value. I've explained this a bit more in https://github.com/daramir/keep-core/blob/47254ca5f2aca3fe615c6e2b2fcd00c5cc859959/solidity/dashboard/README.md#L80 but I see why this doesn't sound great. Should the:
- Readme specify that it's 1337=>3 and that none this changes shouldn't be committed?
- logic of `getChainId` change so that it doesn't depend on NODE_ENV or use a hard-coded value?